dashboard/app/helpers/levels_helper.rb

```diff
@@ -215,6 +215,17 @@ def app_options
       shouldShowDialog: @level.properties['skip_dialog'].blank? && @level.properties['options'].try(:[], 'skip_dialog').blank?
     }
 
+    if current_user
+      section = current_user.sections_as_student.find_by(script_id: @script.try(:id)) ||
```

Comment:
Given that a student can be in multiple sections assigned to the same script... This seems to yield some bad edge case behavior.